ale_linters/ruby/brakeman.vim

```diff
@@ -27,20 +35,22 @@ function! ale_linters#ruby#brakeman#Handle(buffer, lines) abort
 endfunction
 
 function! ale_linters#ruby#brakeman#GetCommand(buffer) abort
+    let l:executable = ale_linters#ruby#brakeman#GetExecutable(a:buffer)
     let l:rails_root = ale#ruby#FindRailsRoot(a:buffer)
 
     if l:rails_root is? ''
         return ''
     endif
 
-    return 'brakeman -f json -q '
+    return ale#Escape(l:executable)
+    \    . ' -f json -q '
     \    . ale#Var(a:buffer, 'ruby_brakeman_options')
     \    . ' -p ' . ale#Escape(l:rails_root)
 endfunction
 
 call ale#linter#Define('ruby', {
 \    'name': 'brakeman',
-\    'executable': 'brakeman',
+\    'executable_callback': 'ale_linters#ruby#brakeman#GetExecutable',
```

Comment:
When the callbacks for `executable_callback` just `return ale#Var(a:buffer, variable_name)`, you can write `executable_calback: ale#VarFunc(variable_name)` instead to get the same result.